### PR TITLE
#133 Create e2e tests

### PR DESCRIPTION
**Pull request by PavelFilo**
added e2e tests:
- login
- create paas offence (check valid plate, check invalid plate, check duplicity, check duplicity modal, check NEV)
- create municipal offence (check duplicity, check duplicity modal, check NEV)

### CODE REVIEW

File: .maestro/create-offence.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    visible: 'Vyberte druh priestupku'
+    timeout: 10000
+- tapOn: 'Vyberte druh priestupku'
+- tapOn: 'Nerešpektovanie DZ'
+- tapOn: Pokračovať
+- runFlow:
+    when:
+      visible: Duplicita
+    commands:
+      - tapOn: Vytvoriť hlásenie
```
Comment (radoslavzeman) [inline .maestro/create-offence.yaml:35]: Is it okay to not have `''` around multiple words string? In other plases, you use `''`, e.g. `- tapOn: 'Nerešpektovanie DZ'`

File: modules/camera/hooks/useScanLicencePlate.ts [hunk 2]
```diff
@@ -74,7 +75,7 @@ export const useScanLicencePlate = () => {
    * Finds the biggest block of text in the frame and checks whether it meets the criteria for ECV
    */
   const scanLicencePlate = useCallback((frameObject: TextData) => {
-    if (!frameObject.blocks) return ''
+    if (!frameObject?.blocks) return ''
```
Comment (radoslavzeman) [inline modules/camera/hooks/useScanLicencePlate.ts:78]: Hmm?

**Review by radoslavzeman — APPROVED** [review]

File: modules/camera/hooks/useScanLicencePlate.ts (hunk 2 above)
Comment (PavelFilo) [reply modules/camera/hooks/useScanLicencePlate.ts:78]: No clue why it came as undefined... the library has the type that says frame object will be there always... but I will make the type not required in the function and write the comment

File: .maestro/create-offence.yaml (hunk 1 above)
Comment (PavelFilo) [reply .maestro/create-offence.yaml:35]: no need to use `''`. I will remove all the occurrences

**Pull request merged**